### ISSUE TITLE
sprite.setPsram(true)  + sprite.setColorDepth(16) 

### ISSUE THREAD

**Issue by tobozo** [lovyan03/LovyanGFX#59]
- Observed on M5Core2 and Wrover-Kit (ILI9341)
- Using this for creating a 320x240 sprite:

```C
    static LGFX_Sprite sprite = LGFX_Sprite( &tft );
```

these work:

```C
    sprite.setColorDepth( 8 );
    sprite.setPsram( true );
```
```C
    sprite.setColorDepth( 24 );
    sprite.setPsram( true );
```

![image](https://user-images.githubusercontent.com/1893754/98468223-cd3ed900-21d9-11eb-8b47-725f61af02ac.png)

however these do not seem to produce a valid result

```C
    sprite.setColorDepth( tft.getColorDepth() );
    sprite.setPsram( true );
```
```C
    sprite.setColorDepth( 16 );
    sprite.setPsram( true );
```
![image](https://user-images.githubusercontent.com/1893754/98468234-d92a9b00-21d9-11eb-8590-e76e8e5136bc.png)

Is that a limitation of the display ?

**Comment by lovyan03** [comment]
@tobozo 
Thanks for reporting this!

I found a flaw in the behavior when using PSRAM.
I'm working on fixing it and will be making a release today.

**Comment by lovyan03** [comment]
@tobozo 
The fix is complete and ver 0.2.6 has been released.
thanks !

**Comment by tobozo** [comment]
fix confirmed, closing this
